### ISSUE TITLE
Improving cv-date-picker 

### ISSUE THREAD

**Issue by yzzhuo** [carbon-design-system/carbon-components-vue#1700]
**Description**
There is an accessibility problem in the cv-date-picker component regarding its use of ARIA roles. Currently, the component assigns role="datepicker" to the input element, but this is not a valid ARIA role according to the WAI-ARIA specification. This results in a critical accessibility violation flagged by tools like Axe.
![Image](https://github.com/user-attachments/assets/f4ebc2e1-8935-417d-b38d-c226e62078e0)

**Expectation**
Ensure the component fully adheres to WCAG 2.2 standards.

**Plans:**
- Update accessibility of the role value:
Suggested by https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/, If necessary, replace it with a valid ARIA role, such as role="combobox", and include proper attributes like aria-expanded, aria-controls.

**Comment by davidnixon** [comment]
I see that the React component does not set an aria role for this component.
However, the rect components also do not follow the advice from w3.org for date pickers. https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/

I'll remove the arai role and watch the react component to see if the make future changes.

**Comment by davidnixon** [comment]
@yzzhuo can you have a look at the PR
[fix: for date picker, remove invalid aria role and add aria-description #1712](https://github.com/carbon-design-system/carbon-components-vue/pull/1712)

**Comment by davidnixon** [comment]
@yzzhuo can you have a look at the PR
Also the fix is deployed here: 
https://davidnixon.github.io/carbon-components-vue/?path=/story/component-cvdatepicker--default
